src/belt_manager.py

```diff
@@ -0,0 +1,205 @@
+                    #add to _potions_remaining count
+                    self.potions_remaining[x] = self.potions_remaining[x] +1
+
+        if toggleBelt:
+            wait(0.2,0.3)
+            keyboard.send(self._config.char["show_belt"]) #toggle belt off
+            wait(0.2,0.3)
+
+                #if belt_contents[x][y] is not self._config.char
+
+    def fill_up_belt_from_inventory(self):
+        """
+        Fill up your belt with pots from the inventory e.g. after death. It will open and close invetory by itself!
+        Updated 11/2021 to respect assigned potion columns
+        :param num_loot_columns: Number of columns used for loot from left
+        """
+        inv_left = self._config.ui_pos["inventory_top_left_slot_x"]
+        inv_top = self._config.ui_pos["inventory_top_left_slot_y"]
+        inv_width = self._config.ui_pos["slot_width"]*self._config.char["num_loot_columns"]
+        inv_height = self._config.ui_pos["slot_height"]*4
+        num_loot_columns=self._config.char["num_loot_columns"]
+        keyboard.send(self._config.char["inventory_screen"])
+        wait(0.7, 1.0)
+        keyboard.press("shift")
+        wait(0.1, 0.15)
+        for potion_slot in range(4):
+            potion_type=self._config.char["belt_slots"][potion_slot]
+            if "healing" in potion_type or "mana" in potion_type:
+                potion_template_key=("inv_super_"+potion_type).upper()
+            elif "rejuvenation" in potion_type:
+                potion_template_key=[("inv_"+potion_type).upper(),("inv_full_"+potion_type).upper()]
+            #Logger.debug(f"potion_slot: {potion_slot} potion_type: {potion_type} potion_template_key: {potion_template_key}")
+            while (self.potions_remaining[potion_slot] < self.belt_height):
+                #self._template_finder.search_and_wait(potion_template_key, img, roi=[inv_left, inv_top, inv_width, inv_height], threshold=0.9,time_out=0.75) TypeError: search_and_wait() got multiple values for argument 'roi'
+                img = self._screen.grab()
+                found, pos = self._template_finder.search(potion_template_key, img, roi=[inv_left, inv_top, inv_width, inv_height], threshold=0.9)
+                if found:
+                    #Logger.debug("found")
+                    x, y = self._screen.convert_screen_to_monitor(pos)
+                    mouse.move(x, y, randomize=1)
+                    wait(0.1, 0.2)
+                    mouse.press(button="left")
+                    wait(0.2, 0.3)
+                    mouse.release(button="left")
+                    self.potions_remaining[potion_slot] = self.potions_remaining[potion_slot] +1
+                else:
+                    #Logger.debug("not found")
+                    break
+                wait(0.1, 0.2)
+        wait(0.5, 0.65)
+        keyboard.release("shift")
+        keyboard.send(self._config.char["inventory_screen"])
+
+
+if __name__ == "__main__":
+    keyboard.wait("f11")
+    #idk what this does
```

Comment:
This part wil be called in case you directly execute `python src/belt_manager.py`. I usually use this for testing.
This code will not be called if you do `from belt_manager import BeltManager`